src/pages/home/components/Sidebar/components/SocialAccordion/SocialAccordion.spec.tsx

```diff
@@ -77,19 +62,26 @@ describe.skip('SocialAccordion', () => {
     it('closes the accordion', async () => {
       render(
         <SocialAccordion
-          accounts={mockList}
+          accounts={mockedAccounts().data}
           error={false}
-          socialMediaId="FACEBOOK_SOCIAL_MEDIA_ID"
+          socialMediaId="DISCORD_EXAMPLE_ID"
         />
       );
+      const [account] = mockedAccounts().data;
 
-      const openAccordion = screen.getByText(/facebook/i);
-      const userCard = screen.getByText(/jhon doe/i);
+      const openAccordion = await screen.findByRole('button');
```

Comment:
aqui não poderia ser accordion 🤔? pq ele não está no estado de aberto sempre
```suggestion
      const accordion = await screen.findByRole('button');
```